src/ape/managers/networks.py

```diff
@@ -303,11 +334,15 @@ def get_provider_from_choice(
                 provider_settings=provider_settings
             )
 
+        elif network_choice.startswith("http://") or network_choice.startswith("https://"):
```

Comment:
thought: I wonder if other protocols should be supported, like `ipc://`.

Reply:
Yes, but that will require some additional effort to `ape-geth` so will save for another time most likely

Reply:
ah, yeah, with web3.py you'd need to use `IPCProvider` and `ape_geth` only supports `HTTPProvider` currently.  :+1: